definitions/install.yaml

```diff
@@ -3058,6 +3058,16 @@ spec:
                             type: string
                         type: object
                     type: object
+                  admissionRequestInfo:
```

Comment:
Please run `make kustomize-crd` to update all CRD manifests.

Reply:
I have added the changes accordingly.